lib/response.ex

```diff
@@ -25,7 +25,7 @@ defmodule HTTPServer.Response do
     %{
       "Content-Length" => "#{String.length(body)}",
       "Content-Type" => "text/plain",
-      "Host" => "127.0.0.1:4000"
+      "Host" => "0.0.0.0:4000"
```

Comment:
This entire commit seems like a separate campsite pr in and of itself. On teams, I suggest, when at all possible, take what you know from the single responsibility principle and apply it to your PRs.  This comes in really handy if you ever need to roll back a PR.

Think about the hypothetical situation where you wanted to rollback these changes to the Host, but keep the refactoring of the handle/1 functions.  Instead of just rolling back the PR you would have to manually crate a new PR and cherry pick this commit. Granted, that is not a lot of work, but imagine if there was more to it, it can get ... messy.

Reply:
^^ this feedback is just for the future, no reason to change this PR.

Reply:
That's helpful to hear! My thought process was that all these changes will help me for the redirect ticket but you're absolutely right, each commit is doing two totally separate things. I'll try to keep that separation in mind for future PRs!